internal/npm_install/npm_install.bzl

```diff
@@ -79,34 +75,57 @@ COMMON_ATTRIBUTES = dict(dict(), **{
         default = True,
         doc = "If stdout and stderr should be printed to the terminal.",
     ),
+    "symlink_node_modules": attr.bool(
+        doc = """Turn symlinking of node_modules on
+        
+        This requires the use of Bazel 0.26.0 and the experimental
+        managed_directories feature.
+        
+        When true, the package manager will run in the package.json folder
+        and the resulting node_modules folder will be symlinked into the
+        external repository create by this rule.
+        
+        When false, the package manager will run in the external repository
+        created by this rule and any files other than the package.json file and
+        the lock file that are required for it to run should be listed in the
+        data attribute.""",
+        default = True,
```

Comment:
Should the behavior of nested node modules be mentioned here as well?

Reply:
Not sure what you mean

Reply:
I thought the comment below was somehow referring to nested node modules. I was totally wrong!
```
# Do not symlink node_modules as when used in downstream repos we should not create
# node_modules folders in the @build_bazel_rules_nodejs external repository. This is
# not supported by managed_directories.
```